magprime/templates/emails/food_interest.txt

```diff
@@ -14,5 +14,4 @@ Thanks!
 
 MAGFest Staff Suite
 chefs@magfest.org
-
-http://magfest.org
+staffsuite.magfe.st
```

Comment:
You might consider making this http://staffsuite.magfe.st so it will clearly be a website, and most likely clickable.